lib/aruba/api.rb

```diff
@@ -8,7 +8,7 @@
 require 'aruba/api/core'
 require 'aruba/api/command'
 
-if Aruba::VERSION <= '1.0.0'
+if Aruba::VERSION <= '1.1.0'
```

Comment:
Yes, might be. Some months ago I started getting rid of them, but somehow I managed this branch to get lost. Since I move my "working environment". But I'm quite eager to remove them as fast as possible. There are tons of (very) old code which is deprecated for ages.